compiler/src/iree/compiler/Codegen/LLVMCPU/test/transform_dialect_bufferize.mlir

```diff
@@ -38,5 +40,4 @@ transform.sequence failures(propagate) {
   transform.iree.eliminate_empty_tensors %variant_op : (!pdl.operation) -> ()
   %variant_op_3 = transform.iree.bufferize %variant_op : (!pdl.operation) -> (!pdl.operation)
   %func = transform.structured.match ops{["func.func"]} in %variant_op_3 : (!pdl.operation) -> !pdl.operation
-  transform.iree.erase_hal_descriptor_type_from_memref %func : (!pdl.operation) -> ()
```

Comment:
I didn't realize this was happening for LLVMCPU as well. Should I move the pass on the CPU side to `addLowerToLLVMPasses` as well?